src/scss/_theme.scss

```diff
@@ -1,7 +1,8 @@
 /// @brand master
 /// @param {String} $theme either b2b or b2c
 /// @outputs Changes navigation colours based on $theme
-@mixin oHeaderServicesTheme($theme) {
+/// @access private
+@mixin _oHeaderServicesTheme($theme) {
```

Comment:
Do we want to allow custom themes? Where `$theme` would be a map of variables? I'd lean toward keeping it private for now if no one is running with a custom header already.

Reply:
By the sounds of the conversation Olga started today, I don't think themes are of particular interest, much less custom themes. I think what _will_ be interesting, is custom behaviour, where we can start adding behavioural branding!!! exciting!!!!!